tests/test_framework/lianad.py

```diff
@@ -107,8 +106,9 @@ def restart_fresh(self, bitcoind):
         self.stop()
         dir_path = os.path.join(self.datadir, "regtest")
         shutil.rmtree(dir_path)
-        wallet_path = os.path.join(dir_path, "lianad_watchonly_wallet")
-        bitcoind.node_rpc.unloadwallet(wallet_path)
+        if BITCOIN_BACKEND_TYPE is BitcoinBackendType.Bitcoind:
```

Comment:
I don't think that would work as `self.backend` is of type `BitcoinBackend` rather than `BitcoinBackendType`.

Another option would be to add a `backend_type()` method to the `BitcoinBackend` abstract class.